virtualenv/builders/base.py

```diff
@@ -44,7 +50,35 @@ def __init__(self, python, flavor, system_site_packages=False,
     def check_available(cls, python):
         raise NotImplementedError
 
+    def _get_base_python_bin(self):
+        bindir = json.loads(
+            check_output([
+                self.python,
+                "-c",
+                textwrap.dedent("""
+                import json
+                import os
+                import sys
+                try:
+                    import sysconfig
+                except ImportError:
+                    from distutils import sysconfig
+
+                if (sys.platform.startswith("win") or sys.platform == "cli" and os.name == "nt"):
+                    bindir = getattr(sys, "real_prefix", sys.prefix)
+                else:
+                    bindir = sysconfig.get_config_var("BINDIR")
+
+                print(json.dumps(bindir))
+                """)
+            ]).decode(locale.getpreferredencoding()),
```

Comment:
You're probably right. My plan is to add integration tests with non-ascii paths later on, we'll see this, and many other places having failures.